benchexec/tools/veriabs.py

```diff
@@ -31,8 +31,8 @@ class Tool(benchexec.tools.template.BaseTool):
         "supportFiles",
     ]
 
-    def executable(self):
-        return util.find_executable("scripts/veriabs")
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("scripts/veriabs")
```

Comment:
```suggestion
        return tool_locator.find_executable("veriabs", subdir="scripts")
```
This will have better behavior if a user decides to add VeriAbs to their `PATH`.